src/api/payloads/UsersPayload.ts

```diff
@@ -0,0 +1,2 @@
+export interface UsersPayload {
+}
```

Comment:
This is where you'd outline the structure of the return payload

Reply:
Creating issue.